templates/docs/base/code.md

```diff
@@ -34,6 +34,21 @@ View example of the base pre block
 View example of the base code block
 </a></div>
 
+### Code Snippets
+
+A code snippet is composed of one or more related components, contained within a `.p-code-snippet` wrapper. Available components are:
```

Comment:
I feel like this sentence makes it feel like code snippet is much more modular than intended.

I wouldn't say 'related components', as 'components' is how we call our higher level building blocks - so 'Code snippet' is a component.
Also it's not intended to just use headers freely. They are optional, but they are meant to be always used with code blocks.

So maybe something along the lines of "The code snippet is used to give more context to code blocks represented on a page. It consist of one or two code blocks with optional headers for their titles."

Reply:
@lyubomir-popov Can you find latest Code snippet spec, so we can make sure our docs reflect the intended uses of it?

Reply:
@bartaz The latest I have is a mention in an email from @wgx - https://miro.com/app/board/o9J_kqQPDak=/?moveToWidget=3074457349016924433&cot=12

Added that to the [brief](https://docs.google.com/document/d/1HwKSetstIGUwZokhaZO6kJDLb1FRDRNQn2tK5td24oI/edit#) as the existing miro boardthere seems seems much older. @wgx Can you confirm this is the latest?

Reply:
@lyubomir-popov The latest is here: https://miro.com/app/board/o9J_kqQPDak=/?moveToWidget=3074457351246118802&cot=14 and in the public spec here: https://discourse.ubuntu.com/t/code-snippet-pattern/19195